src/db/inout/importcsv.cc

```diff
@@ -694,10 +694,10 @@ static std::pair<db_a_elmnt_t, persist::asset_operation>
             bios_throw("element-not-found", id_str.c_str ());
         }
         if ( ids.count(id) == 1 ) {
-            std::string msg = "Element id '";
-            msg += id_str;
-            msg += "' found twice, aborting";
-            bios_throw("bad-request-document", msg.c_str());
+            char *msg = TRANSLATE_ME (zsys_sprintf ("Element id '%s' found twice, aborting", id_str.c_str ()));
```

Comment:
TRANSLATE_ME and zsys_sprintf should be done in a different order

Reply:
Fixed